proj/src/components/statistics/statistics.js

```diff
@@ -1,31 +1,28 @@
 import React, { Component } from 'react'
+import { connect } from 'react-redux';
 import styled from 'styled-components'
 import { flow, map, keys, flatten, uniq, filter } from 'lodash/fp';
 import {BarChart, Bar, XAxis, YAxis, CartesianGrid, Tooltip, Legend} from 'recharts';
 
+import { fetchData } from 'actions/statisticsActions'
+
 const Wrapper = styled.div`
   height: 100vh;
   display: flex;
   justify-content: center;
   align-items: center;
 `
 
-export default class Statistics extends Component {
-
-  state = {
-    data: [
-      {date: 'day 1', Vision: 5, RunTime: 3},
-      {date: 'day 2', Vision: 2, RunTime: 4, Fox: 3},
-      {date: 'day 3', Vision: 3, RunTime: 8, Fox: 4},
-      {date: 'day 4', Vision: 4, RunTime: 5, Fox: 5, Nort: 2},
-      {date: 'day 5', Vision: 11, RunTime: 1, Fox: 6, Barc: 2},
-      {date: 'day 6', Vision: 6, RunTime: 4, Fox: 7},
-      {date: 'day 7', Vision: 7, RunTime: 3, Fox: 10, Nasa: 8}
-    ]
+class Statistics extends Component {
+
+  state = {}
+
+  componentDidMount() {
+    this.props.fetchData()
   }
 
   drawBars(){
-    let data = this.state.data || [];
+
     let dataKeys = []
```

Comment:
const dataKeys = flow([
      map(keys),
      flatten,
      uniq,
      filter((key) => key !== 'date')
    ])(this.props.chartData);